Junk4Dollars-app/Auctions/Auction.swift

```diff
@@ -5,6 +5,25 @@ public struct Auction: Codable {
     public let description: String
     public let starting_price: Int
     public let ends_at: String
+    public var bid: Bid? = nil
```

Comment:
Nice use of an optional here. To me, it communicates the fact that there may not be a bid.